source/extensions/filters/listener/http_inspector/http_inspector.cc

```diff
@@ -43,6 +43,7 @@ Network::FilterStatus Filter::onAccept(Network::ListenerFilterCallbacks& cb) {
               transport_protocol);
     return Network::FilterStatus::Continue;
   }
+  ENVOY_LOG(trace, "http inspector: set transport protocol to raw_buffer");
```

Comment:
I feel the `debug` level is better here.

Reply:
Its not clear to me why this is _setting_ the transport protocol in the log. As far as I know this is just looking at the transport protocol. http_inspector only sets application protocol?

Reply:
Yes, this is wrong.

Reply:
Sure.